app/app/walletButton.tsx

```diff
@@ -29,10 +29,11 @@ export default function WalletButton() {
   return (
     <>
       <a
-        className="font-mono uppercase tracking-widest p-4 border-r text-sm leading-6 bg-primary cursor-pointer"
+    className="flex w-48 px-7 py-3 justify-center items-center gap-2.5 flex-shrink-0 rounded-full bg-[#007D98]
+        text-white text-center font-sans text-sm font-bold leading-5"
         onClick={onClick}
       >
-        {connectedWallets > 0 ? "WALLETS" : "CONNECT WALLET"}
+        {connectedWallets > 0 ? "Wallets" : "Connect Wallet"}
```

Comment:
What do we want to show here precisely?  We can have one wallet connected at a time, we can have multiple previous connected wallets, that wont ask for permission, but here maybe we can show the first 4 and last 4 chars of the wallet if it's connected

Reply:
We can have many wallets connected at the same time, for now it supports multi connection for BTC and ETH wallets.